### PR TITLE
Clean up tests

### PR DESCRIPTION
**Pull request by franmomu**
- Removed some deprecations
- Use Container instead of the mock to simplify

### CODE REVIEW

File: tests/App/config.yml [hunk 1]
```diff
@@ -6,8 +6,10 @@ framework:
         enabled: true
 
 twig:
+    exception_controller: null
```
Comment (phansys) [inline tests/App/config.yml]:
```suggestion
    exception_controller: ~
```

Comment (franmomu) [reply tests/App/config.yml]: I used to use `~`, but I've seen sometimes it is not clear its meaning, so I think using `null` is more straightforward.

Comment (greg0ire) [reply tests/App/config.yml]: @phansys I'll let you merge if you agree, I don't have a strong opinion on this.

Comment (greg0ire) [reply tests/App/config.yml]: @franmomu do you mean the specification says it should be interpreted as null in some contexts, or just that not everybody know that this means `null`?

Comment (franmomu) [reply tests/App/config.yml]: > @franmomu do you mean the specification says it should be interpreted as null in some contexts, or just that not everybody know that this means `null`?

The second one, apparently looks like Symfony were changing `~` to `null` in the recipes https://github.com/symfony/recipes/pull/635

Comment (phansys) [reply tests/App/config.yml]: Let's merge then :+1:

**Comment by phansys** [comment]
Thank you @franmomu!

**Pull request merged**